src/test/java/io/stargate/sgv2/jsonapi/api/v1/VectorSearchIntegrationTest.java

```diff
@@ -588,6 +588,69 @@ public void happyPath() {
 
     @Test
     @Order(3)
+    public void happyPathWithIncludeAll() {
+      String json =
+          """
+                          {
+                            "find": {
+                              "sort" : {"$vector" : [0.15, 0.1, 0.1, 0.35, 0.55]},
+                              "projection" : {"*" : 1},
```

Comment:
Can we add a test case with non-zero and non-one value as well? (to make sure it behaves as true)

Reply:
@kathirsvn Sounds good, will add in suitable place.

Reply:
Added more unit tests to cover non-0/non-1 numbers, booleans.